core/src/navigation/GlobalSearch.html

```diff
@@ -272,6 +293,11 @@
       inputElem,
       luigiCustomSearchRenderer__slot
     });
+
+    if (GenericHelpers.isFunction(search.searchProvider.toggleSearch)) {
+      let fieldVisible = !!isSearchFieldVisible ? !isSearchFieldVisible : true;
+      return search.searchProvider.toggleSearch(inputElem, fieldVisible);
```

Comment:
Do we need `return` here? or it just triggers `toggleSearch` function?

Reply:
change variable to : 

const fieldVisible = isSearchFieldVisible === undefined? true : isSearchFieldVisible;

Yes, we do not need to return ... I'll remove it